### PR TITLE
feat: remove alias in commands and add version injection for cli

### PR DESCRIPTION
**Pull request by lizzy-0323**
## Summary

1. Remove alias in every command
2. Add version injection for cli tool

### CODE REVIEW

File: internal/cli/cmd/version/version.go [hunk 1]
```diff
@@ -14,19 +14,83 @@ See the Mulan PSL v2 for more details.
...
+	Version    = ""
+	CommitHash = ""
+	BuildTime  = ""
 )
 
+// defaultVersionTemplate is the default template for displaying version information.
+const defaultVersionTemplate = `
+OceanBase Operatore Cli:
+ Version:    {{.Version}}
+ OS/Arch:	{{.OS}}/{{.Arch}}
```
Comment (powerfooI) [inline internal/cli/cmd/version/version.go:39]: The key-values seem not to be aligned well.

image

File: make/cli.mk
Comment (powerfooI) [inline make/cli.mk:1]: Comment the make command to add help message in `make help`. Refer to 
```bash
cli-run: ## Run oceanbase-cli in dev mode
    ...
```

**Comment by lizzy-0323** [comment]
The output when using version command are as follows:
image
image

**Review by powerfooI — APPROVED** [review]
LGTM

**Pull request merged**